### PR TITLE
NT-889 - Added possibility to open modal with enter key

### PR DESCRIPTION
**Pull request by eiriko2303**

### CODE REVIEW

File: src/pages/welcome-web/welcome-web.ts [hunk 1]
```diff
@@ -227,8 +227,10 @@ export class WelcomeWebPage {
       event.preventDefault();
       this.markRelativeActivity(1);
     } else if (event.keyCode === this.KEY_ENTER) {
-      event.preventDefault();
-      this.markRelativeInput();
+      if(document.activeElement.tagName.toLowerCase() === 'input') {
```
Comment (reg0) [inline src/pages/welcome-web/welcome-web.ts]: is that enough? I think search field is also an input

Comment (noro5) [reply src/pages/welcome-web/welcome-web.ts]: Actually right now everything can be opened with an Enter, which is a nice feature :) The problems is if we can be sure, that all of that works as expected. If we want to be sure that everything works, then just check class/parent, if we would like to keep that functionality, then OK.

Comment (eiriko2303) [reply src/pages/welcome-web/welcome-web.ts]: The meaning of using the tagName was to not need to change the function again if we add or remove class names.

Comment (eiriko2303) [reply src/pages/welcome-web/welcome-web.ts]: But shoud i then check something other than the tagname or?

**Pull request merged**